cmd/parser/yml.go

```diff
@@ -38,6 +51,17 @@ func ParseKoolYaml(filePath string) (parsed *KoolYaml, err error) {
 	return
 }
 
+// Parse decodes the target kool.yml
+func (y *KoolYaml) Parse(filePath string) (err error) {
+	var parsed *KoolYaml
+	if parsed, err = ParseKoolYaml(filePath); err != nil {
+		return
```

Comment:
As Codecov complained, isn't there a test we can run with a bad formatted file, so we get this error covering hence the line?

Reply:
@fabriciojs yep, done!